_posts/2023-11-01-sr2024-kickstart-and-rules.md

```diff
@@ -0,0 +1,25 @@
+---
+title: SR2024 Kickstart Out Of This World
+---
+
+Student Robotics 2024 was Kickstarted on Saturday 21st October on our livestream. 30 teams from around the UK and Europe joined us as we introduced competitors to the competition, our Kits, and this year’s game! Teams will have six months to build, program, and test a fully autonomous robot, learning invaluable technical and soft skills along the way.
+
+This year's game, Final Frontier, plunges teams into an interstellar battle for dominance among four formidable civilizations, all vying for control of the cosmos. Teams must retrieve asteroids scattered throughout the asteroid field and transport them back to their home planet or, even better, their intergalactic spaceship currently under construction. However, the stakes are high, as within the confines of the asteroid belt lies The Egg, a cosmic time bomb that, if disturbed, could spell chaos. Teams must strategise, navigate the treacherous field, and decide whether to safeguard their own civilization or sabotage their rivals. With asteroids worth points, spaceships to manage, and The Egg lurking, the universe has never been this competitive. May the most astute civilization prevail in this high-stakes cosmic showdown! For complete details, including prizes up for grabs this year, you can consult our [rulebook][rules].
+
+Following on from last year’s success, we’re continuing to offer teams a set of Challenges to be completed throughout the competition year. These encourage teams to start work on their robots early and cover movement, sensors, and vision. They may be approached in any order, and completing challenges before certain deadlines will earn the teams invaluable bonus league points. Full details are available in the rulebook and the deadlines are on our [events page][events].
+
+This year's teams have already been strategising their approaches, and we can't wait to see their progress over the year!
+
+If you weren’t able to attend Kickstart this year, or would like a recap, you can:
+
+- watch our livestream where we explain the game and how this year's competition will work.
+- download [the presentation][presentation] in which we explain this year’s game and how the competition is running this year.
+- download [the microgames][microgames] which are small activities designed to let you become familiar with the kit.
+- If you're a competitor, be sure to check out our [Kickstarted, now what?][now-what] guide for some next steps.
+
+[rules]: https://studentrobotics.org/docs/rules/
```

Comment:
I wonder if we should deep link to the rules for the year rather than this overview page? That would ensure that the link is still correct in a year's time. We do seem to link to this page at least in recent history though :shrug:

Reply:
👍 [a7ddbd8](https://github.com/srobo/website/pull/533/commits/a7ddbd8cf55948274c1c428d7f695653f010daaa)